catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -586,18 +587,36 @@ public Serializable parseDate(Serializable value) {
 
     String string = String.valueOf(value);
 
+    if (StringUtils.isBlank(string)) {
+      return null;
+    }
+
+    if (StringUtils.isNumeric(string)) {
+      try {
+        return Instant.ofEpochMilli(Long.parseLong(string));
+      } catch (NumberFormatException ex) {
+        throw new IllegalArgumentException(ex);
```

Comment:
❓ Any thoughts on aligning how we do error control here? Specifically, 591 & 584 we return null but here we throw an exception. Should not understanding the type be treated the same as not being able to understand something that looks like a number? Also it looks like on 621 we throw an exception if we can't parse. 

For this method in general does it make sense to make it more lenient and have it return null in the failure cases?(especially if this is a common point of failure and we would prefer to not have this halt data getting into the system)